Issues getting metrics to run.

I've previously used Tachometer on my component library with code inspired heavily by that found in the Material Web Components project and had no trouble getting the packages to install, build or load. When recently working to update to v0.5.1, when attempting to run tests I get the following dump:
```
/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript/lib/typescript.js:19871
            var sourceFile = new SourceFileConstructor(290 /* SourceFile */, /*pos*/ 0, /* end */ sourceText.length);
                                                                                                             ^

TypeError: Cannot read property 'length' of undefined
    at createSourceFile (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript/lib/typescript.js:19871:110)
    at Object.parseJsonText (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript/lib/typescript.js:19709:26)
    at Object.parseJsonText (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript/lib/typescript.js:19530:23)
    at Object.parseConfigFileTextToJson (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript/lib/typescript.js:27974:33)
    at programFromConfig (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript-json-schema/dist/typescript-json-schema.js:1063:21)
    at Object.exec (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript-json-schema/dist/typescript-json-schema.js:1098:19)
    at Object.run (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript-json-schema/dist/typescript-json-schema-cli.js:50:30)
    at Object.<anonymous> (/Users/westbrook/Documents/web/repos/swc/node_modules/tachometer/node_modules/typescript-json-schema/bin/typescript-json-schema:3:5)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tachometer@0.5.1 generate-json-schema: `typescript-json-schema tsconfig.json ConfigFile --include src/configfile.ts --required --noExtraProps > config.schema.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tachometer@0.5.1 generate-json-schema script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/westbrook/.npm/_logs/2020-09-01T00_57_58_773Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tachometer@0.5.1 build: `rimraf lib/ client/lib/ && mkdir lib && npm run generate-json-schema && tsc && tsc -p client/ && npm run lint`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tachometer@0.5.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/westbrook/.npm/_logs/2020-09-01T00_57_58_788Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
Error: Command failed: yarn add chromedriver@*
    at makeError (/Users/westbrook/Documents/web/repos/swc/node_modules/execa/index.js:174:9)
    at /Users/westbrook/Documents/web/repos/swc/node_modules/execa/index.js:278:16
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 1,
  stdout: null,
  stderr: null,
  failed: true,
  signal: null,
  cmd: 'yarn add chromedriver@*',
  timedOut: false,
  killed: false
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I have no problem running tests when reverting to v0.4.2 and am not sure whether there is something that I need to do locally to get the newest version working.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "tachometer": "bin/tach.js"
   },
   "scripts": {
-    "prepare": "npm run build",
+    "prepare": "if [ -f './tsconfig.json' ]; then npm run build; fi;",
     "build": "rimraf lib/ client/lib/ && mkdir lib && npm run generate-json-schema && tsc && tsc -p client/ && npm run lint",
     "generate-json-schema": "typescript-json-schema tsconfig.json ConfigFile --include src/configfile.ts --required --noExtraProps > config.schema.json",
     "lint": "tslint --project . --format stylish",

